static/app/utils/profiling/renderers/flamegraphRenderer2D.tsx

```diff
@@ -13,12 +14,6 @@ import {
 } from 'sentry/utils/profiling/renderers/flamegraphRenderer';
 import {Rect} from 'sentry/utils/profiling/speedscope';
 
-function colorComponentsToRgba(color: number[]): string {
```

Comment:
`profiling/colors/utils` has an exported util for this, but we also had 2 other identical implementations of this util.